src/binary/utils/signing-client.ts

```diff
@@ -30,6 +30,18 @@ export async function signingClient(
   const options: IbcClientOptions = {
     prefix: chain.prefix,
     gasPrice: GasPrice.fromString(chain.gas_price),
+    gasLimits: {
```

Comment:
Looks good. Can you add a test that this works?

Like setting these very low in a test config, and asserting that running a command leads to an error. This would ensure they are actually used.